css/style.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap');
+
+/* Default Styles */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+a {
+  color: #000;
+  text-decoration: none;
+}
+
+/* HEADER STYLES */
+header {
+  width: 100vw;
+  height: 100vh;
+  padding: 10px 40px;
+  background-image: url(../images/header-shapes\ mobile-bg.jpg);
+  background-size: contain;
```

Comment:
- [ ] For the `background-image` I think it's better if you change `background-size` from `contain` to `cover` and it's better to add `background-position` to match the design better 👍 

This is what I've done 👇 
![image](https://user-images.githubusercontent.com/50721479/212066331-5878741e-56ae-4c4e-919f-b21cc42c4c80.png)

And I find the results match the design very well 💯 

## Before:
![image](https://user-images.githubusercontent.com/50721479/212066471-c28987c3-e278-47d0-ad65-701406c94eb3.png)

## After:
![image](https://user-images.githubusercontent.com/50721479/212066438-af283ec8-abef-48bd-88bb-3cc491f59f0d.png)